client/app/components/packages/landuse-form/proposed-actions.hbs

```diff
@@ -1,6 +1,91 @@
 {{#let @form as |form|}}
   <form.Section @title="Proposed Actions">
     <div data-test-section="proposed-actions">
+      {{#if this.projectHasRequiredActions}}
+        <h3>Zoning Special Permits, Authorizations, and Certifications</h3>
+        <p>The following questions relate to Zoning Special Permits, Authorizations and Certifications and the following Actions: [ZA, ZR].</p>
+
+        <Ui::Question as |Q|>
+          <Q.Legend>
+            Is the applicant any of the following?
+            <small class="text-weight-normal">(select all that apply)</small>
+          </Q.Legend>
+
+          <ul class="no-bullet no-margin">
+            {{#each (array
+              (hash attr="dcpOwnersubjectproperty" label="Owner of subject property")
+              (hash attr="dcpLeesseesubjectproperty" label="Lessee of subject property")
+              (hash attr="dcpLeaseorbuy" label="In a contract to lease or buy the subject property")
+              (hash attr="dcpIsother" label="Other (explain in attached project description)"))  as |type|
+            }}
+              <li>
+                <form.Field
+                  @attribute={{type.attr}}
+                  @type="checkbox"
+                  as |Checkbox|
+                >
+                  <Checkbox>
+                    {{type.label}}
+                  </Checkbox>
+                </form.Field>
+              </li>
+            {{/each}}
+          </ul>
+        </Ui::Question>
+
+        <Ui::Question
+          class="no-margin-top"
+          data-test-other-parties-radio-group
+          as |Q|
+        >
+
+        <Q.Legend>
+          Are there other owners or long-term lessees of the subject property? 
+        </Q.Legend>
+
+          <form.Field
+            @attribute="dcpOtherparties"
+            @type="radio"
+            as |RadioButton|
+          >
+            <RadioButton
```

Comment:
Just curious: can you use the radio-group field type here, so you don't have to manually write each radio button?

Reply:
Not sure, it seems like we've been doing this a lot for radio buttons that are just booleans and don't have optionsets?